icu4j/main/classes/core/src/com/ibm/icu/impl/number/LongNameHandler.java

```diff
@@ -298,9 +301,38 @@ public MicroProps processQuantity(DecimalQuantity quantity) {
         return micros;
     }
 
+    // FIXME: consider name, style, etc
```

Comment:
If the name/style/etc was considered good, delete the FIXME. (Yesterday I was rushing out contributed fixes and workarounds, not polishing the PR.)

Reply:
Done: fixed the comment with c39ab4eaf2, having implemented the interface with 15e46ba6cd.